truffle/com.oracle.truffle.api.test/src/com/oracle/truffle/api/vm/EngineTest.java

```diff
@@ -117,4 +118,19 @@ public void wrappedAsArray() throws Exception {
         assertEquals(2, arr[1].intValue());
         assertEquals(3, arr[2].intValue());
     }
+
+    @Test
+    public void initializePolyglotEngineWithArguments() throws IOException {
+        PolyglotEngine vm = createBuilder().setArguments("application/x-test-import-export-1", new String[]{"1", "2"}).build();
+        PolyglotEngine.Language language1 = vm.getLanguages().get("application/x-test-import-export-1");
+
+        // TODO: remove once initialization issue is solved for
+        // https://github.com/graalvm/truffle/pull/9
+        language1.eval(Source.fromText("return=arr", "get the array")).as(AccessArray.class);
+
+        Env env = language1.getEnv(true);
```

Comment:
Isn't this broken? I am getting exceptions from here. I suggest to use my version of the test - it uses just public methods.